pageserver/src/layered_repository.rs

```diff
@@ -1099,6 +1099,9 @@ pub struct LayeredTimeline {
     // It can be unified with latest_gc_cutoff_lsn under some "first_valid_lsn",
     // though lets keep them both for better error visibility.
     initdb_lsn: Lsn,
+
+    /// Current physical size of the timeline at the lastest LSN
+    physical_size: AtomicU64,
```

Comment:
Worth noting which kind of files we actually take into account. E g we do not count temp download files, metadata, etc

Reply:
Yeah, this variable is constructed based on the files in `layer_map`. So it won't take into account temp file, metadata.